lib/active_merchant/billing/gateways/moneris.rb

```diff
@@ -33,7 +33,8 @@ def initialize(options = {})
         requires!(options, :login, :password)
         @cvv_enabled = options[:cvv_enabled]
         @avs_enabled = options[:avs_enabled]
-        options = { :crypt_type => 7 }.merge(options)
+        @cof_enabled = options[:cof_enabled]
+        options = {:crypt_type => 7}.merge(options)
```

Comment:
I get what's going on here, and I know it's not your code or anything, but this is a _ton_ slower (and wastes way more memory) than just `options[:crypt_type] = 7 if !options.has_key?(:crypt_type)`. _If you're back in here for some other reason_, might be worth switching it.

Reply:
Sure, no problem.